lib/components/molecules/Accordion/Accordion.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+/**
+ *
+ * Accordion
+ *
+ */
+import React, { useContext } from 'react';
+import type { Node } from 'react';
+import styled from 'styled-components';
+import styles from './Accordion.style';
+import type { AccordionHeaderProps, AccordionPanelProps, AccordionProps } from './types';
+
+const AccordionContext = React.createContext('');
+const AccordionPanel = ({ children, className, id }: AccordionPanelProps): Node => (
+  <AccordionContext.Provider value={id}>
+    <details className={className} id={id}>
+      {children}
+    </details>
+  </AccordionContext.Provider>
+);
+
+AccordionPanel.defaultProps = {
+  className: 'accordion-panel',
+};
+
+const AccordionHeader = ({ children, className, ariaLabelAttr }: AccordionHeaderProps): Node => {
+  const accordionContext = useContext(AccordionContext);
+
+  return (
+    <summary
+      className={className}
+      role="button"
+      aria-expanded="false"
```

Comment:
aria-expanded="false" should change to aria-expanded="true" when its in open state